packages/iov-ethereum/src/ethereumcodec.ts

```diff
@@ -82,7 +90,51 @@ export const ethereumCodec: TxCodec = {
     throw new Error(`Not implemented tx: ${signed}`);
   },
   parseBytes: (bytes: PostableBytes, chainId: ChainId): SignedTransaction => {
-    throw new Error(`Not implemented bytes: ${bytes}, chainId: ${chainId}`);
+    const json = JSON.parse(Encoding.fromUtf8(bytes));
+    // signature
+    const r = Encoding.fromHex(json.r.replace("0x", ""));
+    const s = Encoding.fromHex(json.s.replace("0x", ""));
+    const v = decodeHexQuantity(json.v);
+    const signature = new Uint8Array([...r, ...s, v]) as SignatureBytes;
```

Comment:
The binary signature should be fixed length encoding of extended secp256k1 signature again.

```ts
 const signature = new ExtendedSecp256k1Signature(r, s, recoveryParam).toFixedLength() as SignatureBytes;
```

so we need a reverse operation to `eip155V`

Reply:
Done